### DISCUSSION TITLE
Phishing Websites Takedown Request

### DISCUSSION THREAD

**Discussion by ariana-bajrami** [vercel/community#7305] [Help] [answered]
### Summary

Hello,The following malicious websites, created on your platform, are clearly impersonating Pancakeswap and conducting a phishing attacks: 

[REDACTED]

Considering its deliberately abusive nature, this web threat should be taken down immediately. We have already reported to abuse@vercel.comPlease escalate this and speed up the resolution of the case.Thank you.

### Example

[REDACTED]

### Steps to Reproduce

...

**Comment by pawlean** [comment, marked as answer]
Thank you for reporting a potentially malicious site to us. We appreciate your concern for the safety of our users. Since you've already sent your report to [abuse@vercel.com](mailto:abuse@vercel.com), please feel free to follow up on your open case via email if you have any further questions or additional information to share.

[a comment by an automated account is not shown]
